services/QuillLMS/spec/support/system.rb

```diff
@@ -4,7 +4,7 @@
   Capybara.register_driver :local_selenium_chrome_headless do |app|
     options = Selenium::WebDriver::Chrome::Options.new(
       args: [
-        'headless',
+        # 'headless',
```

Comment:
Should we just remove this line altogether if we're commenting it out?

Reply:
Whoops.  That's a mistake.  I will add that back in.  Thank you for spotting